### ISSUE TITLE
Remove brand_color_2 from database

### ISSUE THREAD

**Issue by mirie** [forem/forem#18434]
## Problem Statement

We [had 2 brand color fields](https://github.com/forem/forem/issues/17686) in /settings/profile but we are using only one which is causing confusion. We updated the display on the profile page with [this PR](https://github.com/forem/forem/pull/18370), but the database migration is still a todo task to wrap up this work.

**Comment by mirie** [comment]
@fdocr can you summarize what we talked about yesterday re: organizations question when you have a chance?

**Comment by fdocr** [comment]
Absolutely, and sorry for the delay @mirie.

I was looking back into the problem I mentioned to you and realized that field isn't shared between both users & organizations. They have the same name on the UI (`/settings/profile` & `/settings/organization/<id>`) and the way the decorator was used in the code tripped me up.

I hope I can get #18473 ready for review soon and merged to close this issue.
